airflow/models/dagbag.py

```diff
@@ -67,13 +68,23 @@
 
 
 class FileLoadStat(NamedTuple):
-    """Information about single file."""
+    """
+    Information about single file.
+
+    :param file: Loaded file.
+    :param duration: Time spent on process file.
+    :param dag_num: Total number of DAGs loaded in this file.
+    :param task_num: Total number of Tasks loaded in this file.
+    :param dags: DAGs names loaded in this file.
+    :param warnings: Total warnings captured during the process file.
```

Comment:
```suggestion
    :param warnings: Count of warnings captured during the process file.
```

Reply:
Why not? Let me change also the name of variable